client/state/action-types.js

```diff
@@ -517,6 +517,8 @@ export const READER_TAGS_REQUEST = 'READER_TAGS_REQUEST';
 export const READER_TAGS_RECEIVE = 'READER_TAGS_RECEIVE';
 export const READER_FOLLOW_TAG_REQUEST = 'READER_FOLLOW_TAG_REQUEST';
 export const READER_FOLLOW_TAG_RECEIVE = 'READER_FOLLOW_TAG_RECEIVE';
+export const READER_RECORD_FOLLOW = 'READER_RECORD_FOLLOW';
```

Comment:
i'm taking over the READER_FOLLOW and READER_UNFOLLOW action. While we're in flux (heh). move the "record" actions to a new name.